packages/ui/css/base.ts

```diff
@@ -46,6 +46,11 @@ html {
     letter-spacing: 0.05em;
     color: ${props => props.theme.colors.grey[700]};
   }
+
+  h4.titleCentred {
+  margin-bottom: 1rem;
+  text-align: center;
+  }
```

Comment:
We should probably not do any global classes like this as we're moving away from that style of dev. Mind if we turn this into a component instead?